tracer/src/Datadog.Trace/Propagators/PropagationContext.cs

```diff
@@ -13,10 +16,20 @@ internal readonly struct PropagationContext
 
     public readonly Baggage? Baggage;
 
+    public readonly IEnumerable<SpanLink>? Links;
+
     public PropagationContext(SpanContext? spanContext, Baggage? baggage)
     {
         SpanContext = spanContext;
         Baggage = baggage;
+        Links = Enumerable.Empty<SpanLink>();
+    }
+
+    public PropagationContext(SpanContext? spanContext, Baggage? baggage, IEnumerable<SpanLink>? extractionSpanLinks)
+    {
+        SpanContext = spanContext;
+        Baggage = baggage;
+        Links = extractionSpanLinks ?? Enumerable.Empty<SpanLink>();
```

Comment:
```suggestion
        Links = extractionSpanLinks ?? [];
```

Reply:
I've allowed this to be `null` in 9b8ccc7466fa356c69cad26d1d6a42a2e21a9018